app/assets/stylesheets/custom.scss

```diff
@@ -134,15 +141,43 @@ a {
   margin-bottom: 54px;
 }
 
+// tables
 td {
   background-color: rgba(255, 255, 255, .5);
 }
 
+th {
+  text-align: center;
+}
+
+// complete loan message
+#complete-loan {
+  font-size: 20px;
```

Comment:
Properties should be ordered display, font-size, text-decoration, text-decoration-color